src/components/books/addNewBook/AddNewBook.js

```diff
@@ -0,0 +1,37 @@
+import React, { useState } from 'react';
+import { v4 as uuidv4 } from 'uuid';
+import { useDispatch } from 'react-redux';
+import { createBook } from '../../../redux/books/books';
+
+const AddNewBook = () => {
+  const dispatch = useDispatch();
+  const [bookInfo, setBookInfo] = useState({
+    item_id: '',
+    title: '',
+    category: '',
+  });
```

Comment:
- [ ] The `useState` is not showing anything after creating a new book, also the `completed`  logo is too big.


![screencapture-localhost-3000-2021-09-30-12_09_33](https://user-images.githubusercontent.com/61562626/135445808-684e9854-3c97-482d-8ff0-26d92a5f50c2.png)

Reply:
Fixed